pkg/api/pod/util.go

```diff
@@ -19,11 +19,12 @@ package pod
 import (
 	"strings"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
not sure why this happend, have to teach my editor to stop doing this

- [ ] @kwiesmueller remove this change